samcli/lib/build/app_builder.py

```diff
@@ -144,13 +144,14 @@ def update_template(self, template_dict, original_template_path, built_artifacts
         return template_dict
 
     def _build_function(self, function_name, codeuri, runtime):
-        config = get_workflow_config(runtime)
 
         # Create the arguments to pass to the builder
```

Comment:
This comment is floating. This seems like docstring worthy

Reply:
done